xyz-psql-connector/src/main/java/com/here/xyz/psql/DatabaseWriter.java

```diff
@@ -432,4 +446,90 @@ private static void fillFailList(int[] batchResult, List<FeatureCollection.Modif
             if (batchResult[i] == 0)
                 fails.add(new FeatureCollection.ModificationFailure().withId(idList.get(i)).withMessage(getFailedRowErrorMsg(action, event)));
     }
+
+    private static SQLQuery buildTruncateSpaceQueryKeepVersionSeq(ModifyFeaturesEvent event, String schema) {
```

Comment:
We need to put this into an own QueryRunner e.g., `EraseSpace`, as the DatabaseWriter is subject to removal soon.